app/views/search_form.php

```diff
@@ -146,8 +157,7 @@ class="search_options"
                                class="search_options"
                                <?=Utils::checkboxState($check['each_word'])?>
                         />
-                        <label for="each_word" aria-describedby="tooltip_eachword">Each Word</label>
-                        <div id="tooltip_eachword" class="search_tooltip" role="tooltip">If selected, search for <strong>each word</strong> in the search query (at least 2 characters long). By default, the search query is used as a whole.</div>
+                        <label>Each Word</label>
```

Comment:
Should be `<label for="each_word">Each Word</label>` (can’t be checked otherwise)